storage_service/locations/models/package.py

```diff
@@ -941,12 +946,32 @@ def start_reingest(self, pipeline, reingest_type):
         elif reingest_type == self.METADATA_ONLY:
             reingest_files.append(os.path.join(relative_path, 'data', 'objects', 'metadata', ''))
 
+        # Fetch processing configuration, put it in the root of the package and
+        # include the file in reingest_files.
+        if reingest_type == self.FULL and processing_config != 'default':
+            try:
+                config = pipeline.get_processing_config(processing_config)
```

Comment:
I like that idea!  I think it's out of scope for right now though. The processing-config endpoint is added in artefactual/archivematica#313 so should be exposed in qa/1.x's dashboard API.